packages/wxt/src/core/builders/vite/plugins/wxtPluginLoader.ts

```diff
@@ -53,18 +70,20 @@ try {
 
         const { document } = parseHTML(html);
         const existing = document.querySelector(`script[src='${src}']`);
-        if (existing) return;
 
-        const script = document.createElement('script');
-        script.type = 'module';
-        script.src = src;
+        if (!existing) {
+          const script = document.createElement('script');
+          script.type = 'module';
+          script.src = src;
+
+          if (document.head === null) {
```

Comment:
Please revert unless it's important to the PR.

```suggestion
          if (document.head == null) {
```